__tests__/Guides.unit.test.ts

```diff
@@ -1,10 +1,388 @@
+import Gesto from 'gesto';
+import { defaultProps, GUIDE } from '../src/react-guides/consts';
 import Guides from '../src/react-guides/Guides';
-import { GuidesProps } from '../src/react-guides/types';
+import { GuidesProps, GuidesState, LockGuides } from '../src/react-guides/types';
+import Ruler, { RulerProps } from '../src/react-ruler';
+
+const defaultGuides: number[] = [];
+const indexOfDeletedGuide = 2;
+let guidesTest: number[] = [5, -6, 3, -4.2];
+const deletedGuide = guidesTest[indexOfDeletedGuide];
+let selectedGuide: number = guidesTest[1];
+let eventMouse: MouseEvent = new MouseEvent('click');
+let isDragging = false;
 
 // Guides
 describe('Guides', () => {
-    it('should be defined', () => {
-        const guide = new Guides({} as Required<GuidesProps>);
-        expect(guide).toBeDefined();
+  let guidesInstance: Guides | null = createInstance();
+  beforeEach(() => {
+    initVariables();
+    guidesInstance = createInstance();
+  });
+
+  afterEach(() => {
+    guidesInstance = null;
+  });
+
+  it('should be defined', () => {
+    expect(guidesInstance).toBeDefined();
+  });
+
+  it('should get ruler props', () => {
+    expect(guidesInstance?.['rulerProps']).toBeInstanceOf(Object);
+  });
+  it('should be get guides', () => {
+    guidesInstance?.getGuides();
+    expect(guidesInstance?.state.guides).toEqual(defaultGuides);
+  });
+
+  it('should be loaded guides', () => {
+    guidesInstance?.loadGuides(guidesTest);
+    expect(guidesInstance?.getGuides()).toEqual(guidesTest);
+  });
+
+  it('should be cleared all guides', () => {
+    guidesInstance?.loadGuides(guidesTest);
+    guidesInstance?.clearAllGuides();
+    expect(guidesInstance?.getGuides()).toEqual(defaultGuides);
```

Comment:
it is not obvious how should default guides looks.
Especially when you rewrite variables globally.
I suspect if you clear all guides it should be like empty array `[]`
So writing like this is more readable and easy to understand what is going on
`expect(guidesInstance?.getGuides()).toEqual([]);`